src/types.ts

```diff
@@ -0,0 +1,32 @@
+export interface LocationResponse {
```

Comment:
Typescript newbie question: why don't you just put these in `declarations.d.ts`?

Reply:
You would not import a `d.ts`. A `d.ts` is either included via `/// ref` or implicitly. For internal code / no module declarations you'd want to use `.ts` as its part of your code and you want it to be explicit. Benefit is that this way TypeScript performs the transpilation as it should, resulting in correct `d.ts` files in the desired output directory.

The only reason to explicitly write `d.ts` files is for declaring ambient modules.

Reply:
I will need to do some more reading, but that's helpful, thank you!